feat: Replace gsoc redirection with actual contents for 2023 (#6829)
jenkins-infra/jenkins.io: content/projects/gsoc/2023/index.adoc

---
layout: project
title: "Google Summer of Code in Jenkins"

> description: "Landing page for the Google Summer of Code program in the Jenkins project"

Reviewer: Since this is for 2023, I think it would be better to include 2023 in the page description
```suggestion
description: "Google Summer of Code 2023 in the Jenkins project"
```

section: projects
tags:
- gsoc2023
sig: gsoc
opengraph:
  image: /images/gsoc/opengraph.png
links:
  gitter: "jenkinsci_gsoc-sig:gitter.im"
  meetings: "/projects/gsoc/#office-hours"
---

// image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=left]
link:https://developers.google.com/open-source/gsoc/[Google Summer of Code]

> (GSoC) is a global, online program focused on bringing students and new contributors into open source software development. GSoC Contributors work with an open source organization on a 10-22 weeks programming project under the guidance of mentors.

Reviewer: We generally prefer sentence per line formatting so that it is easier to review.  Optional
```suggestion
(GSoC) is a global, online program focused on bringing students and new contributors into open source software development.
GSoC contributors work with an open source organization on a 10-22 week programming project under the guidance of mentors.
```

> GSoC contributors accepted into the program receive a stipend and are sponsored by Google, to work on well-defined projects to improve or enhance the Jenkins project.

Reviewer: Minor punctuation change
```suggestion
GSoC contributors accepted into the program receive a stipend and are sponsored by Google to work on well-defined projects to improve or enhance the Jenkins project.
```

> In exchange, numerous Jenkins community members volunteer as "mentors" for the selected GSoC contributors to help integrate them into the open source community and succeed in completing their projects.

Reviewer: ```suggestion
In exchange, numerous Jenkins community members volunteer as mentors for the selected GSoC contributors.
Mentors help the contributors integrate into the open source community and help them succeed in completing their projects.
```

image:/images/gsoc/opengraph.png[Jenkins GSoC, role=center, float=center]

== GSoC 2023

> We have been accepted as a Google Summer of Code mentoring org for 2023. +

Reviewer: No need for the line continuation character.
```suggestion
We have been accepted as a Google Summer of Code mentoring org for 2023.
```
